kubernetes/kustomize/certbot/buildconfig.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
+kind: BuildConfig
+metadata:
+  name: certbot
+spec:
+  completionDeadlineSeconds: 300
+  output:
+    to:
+      kind: ImageStreamTag
+      name: certbot:latest
+  postCommit: {}
+  resources:
+    requests:
+      cpu: 1
+    limits:
+      cpu: 1
+  source:
+    contextDir: docker
+    git:
+      ref: feature/cert-per-host
+      uri: https://github.com/WadeBarnes/certbot.git
```

Comment:
This can be switched back to the [BCDevOps/certbot](https://github.com/BCDevOps/certbot.git) repo.  The features I added were merged.

Reply:
Updated.